### DISCUSSION TITLE
The coordinate axis system is weird.

### DISCUSSION THREAD

**Discussion by CritickalGames** [godotengine/godot-proposals#10230] [Engine Core]
Maybe it's just me, but, i think that could be better if the origin of 2d nodos(i don't know how it is on 3d) are not on top left anymore, and started to be on buttom left. In this way, Z top is not -Z. That's weird.

**Comment by Calinou** [comment]
Thanks for the proposal! Consolidating in https://github.com/godotengine/godot-proposals/issues/2070.

Note that there are no plans to change Godot's coordinate systems to reverse or reorder any axes, as per this comment: https://github.com/godotengine/godot-proposals/issues/2070#issuecomment-2217653180
